commit 639087dd492090ab55e91c43e1e8e62819cb1135 (refs/changes/35/5235/4)
Author: Tim Foster <tim.foster@joyent.com>
Date:   2019-01-08T12:44:04+00:00 (9 months ago)
    
    TOOLS-2133 MG needs to look for .tar.gz agent tarballs after TOOLS-2063

diff --git a/Makefile b/Makefile
index 7042bf7..bcd9c3b 100644
--- a/Makefile
+++ b/Makefile
@@ -126,7 +126,7 @@ clean_smartlogin:
 #---- amon
 
 _amon_stamp=$(SDC_AMON_BRANCH)-$(TIMESTAMP)-g$(SDC_AMON_SHA)
-AMON_BITS=$(BITS_DIR)/amon/amon-pkg-$(_amon_stamp).tar.bz2 \
+AMON_BITS=$(BITS_DIR)/amon/amon-pkg-$(_amon_stamp).tar.gz \
 	$(BITS_DIR)/amon/amon-relay-$(_amon_stamp).tgz \
 	$(BITS_DIR)/amon/amon-agent-$(_amon_stamp).tgz
 AMON_BITS_0=$(shell echo $(AMON_BITS) | awk '{print $$1}')
@@ -179,7 +179,7 @@ clean_amon:
 #   for env setup. Might be demons in there. (RELENG-192)
 
 _ca_stamp=$(SDC_CLOUD_ANALYTICS_BRANCH)-$(TIMESTAMP)-g$(SDC_CLOUD_ANALYTICS_SHA)
-CA_BITS=$(BITS_DIR)/ca/ca-pkg-$(_ca_stamp).tar.bz2 \
+CA_BITS=$(BITS_DIR)/ca/ca-pkg-$(_ca_stamp).tar.gz \
 	$(BITS_DIR)/ca/cabase-$(_ca_stamp).tar.gz \
 	$(BITS_DIR)/ca/cainstsvc-$(_ca_stamp).tar.gz
 CA_BITS_0=$(shell echo $(CA_BITS) | awk '{print $$1}')
@@ -234,7 +234,7 @@ clean_ca:
 
 
 _ufds_stamp=$(SDC_UFDS_BRANCH)-$(TIMESTAMP)-g$(SDC_UFDS_SHA)
-UFDS_BITS=$(BITS_DIR)/ufds/ufds-pkg-$(_ufds_stamp).tar.bz2
+UFDS_BITS=$(BITS_DIR)/ufds/ufds-pkg-$(_ufds_stamp).tar.gz
 UFDS_IMAGE_BIT=$(BITS_DIR)/ufds/ufds-zfs-$(_ufds_stamp).zfs.gz
 UFDS_MANIFEST_BIT=$(BITS_DIR)/ufds/ufds-zfs-$(_ufds_stamp).imgmanifest
 
@@ -278,7 +278,7 @@ clean_ufds:
 #---- ASSETS
 
 _assets_stamp=$(SDC_ASSETS_BRANCH)-$(TIMESTAMP)-g$(SDC_ASSETS_SHA)
-ASSETS_BITS=$(BITS_DIR)/assets/assets-pkg-$(_assets_stamp).tar.bz2
+ASSETS_BITS=$(BITS_DIR)/assets/assets-pkg-$(_assets_stamp).tar.gz
 ASSETS_IMAGE_BIT=$(BITS_DIR)/assets/assets-zfs-$(_assets_stamp).zfs.gz
 ASSETS_MANIFEST_BIT=$(BITS_DIR)/assets/assets-zfs-$(_assets_stamp).imgmanifest
 
@@ -317,7 +317,7 @@ clean_assets:
 #---- ADMINUI
 
 _adminui_stamp=$(ADMINUI_BRANCH)-$(TIMESTAMP)-g$(ADMINUI_SHA)
-ADMINUI_BITS=$(BITS_DIR)/adminui/adminui-pkg-$(_adminui_stamp).tar.bz2
+ADMINUI_BITS=$(BITS_DIR)/adminui/adminui-pkg-$(_adminui_stamp).tar.gz
 ADMINUI_IMAGE_BIT=$(BITS_DIR)/adminui/adminui-zfs-$(_adminui_stamp).zfs.gz
 ADMINUI_MANIFEST_BIT=$(BITS_DIR)/adminui/adminui-zfs-$(_adminui_stamp).imgmanifest
 
@@ -357,7 +357,7 @@ clean_adminui:
 #---- REDIS
 
 _redis_stamp=$(SDC_REDIS_BRANCH)-$(TIMESTAMP)-g$(SDC_REDIS_SHA)
-REDIS_BITS=$(BITS_DIR)/redis/redis-pkg-$(_redis_stamp).tar.bz2
+REDIS_BITS=$(BITS_DIR)/redis/redis-pkg-$(_redis_stamp).tar.gz
 REDIS_IMAGE_BIT=$(BITS_DIR)/redis/redis-zfs-$(_redis_stamp).zfs.gz
 REDIS_MANIFEST_BIT=$(BITS_DIR)/redis/redis-zfs-$(_redis_stamp).imgmanifest
 
@@ -397,7 +397,7 @@ clean_redis:
 #---- amonredis
 
 _amonredis_stamp=$(SDC_AMONREDIS_BRANCH)-$(TIMESTAMP)-g$(SDC_AMONREDIS_SHA)
-AMONREDIS_BITS=$(BITS_DIR)/amonredis/amonredis-pkg-$(_amonredis_stamp).tar.bz2
+AMONREDIS_BITS=$(BITS_DIR)/amonredis/amonredis-pkg-$(_amonredis_stamp).tar.gz
 AMONREDIS_IMAGE_BIT=$(BITS_DIR)/amonredis/amonredis-zfs-$(_amonredis_stamp).zfs.gz
 AMONREDIS_MANIFEST_BIT=$(BITS_DIR)/amonredis/amonredis-zfs-$(_amonredis_stamp).imgmanifest
 
@@ -437,7 +437,7 @@ clean_amonredis:
 #---- RABBITMQ
 
 _rabbitmq_stamp=$(SDC_RABBITMQ_BRANCH)-$(TIMESTAMP)-g$(SDC_RABBITMQ_SHA)
-RABBITMQ_BITS=$(BITS_DIR)/rabbitmq/rabbitmq-pkg-$(_rabbitmq_stamp).tar.bz2
+RABBITMQ_BITS=$(BITS_DIR)/rabbitmq/rabbitmq-pkg-$(_rabbitmq_stamp).tar.gz
 RABBITMQ_IMAGE_BIT=$(BITS_DIR)/rabbitmq/rabbitmq-zfs-$(_rabbitmq_stamp).zfs.gz
 RABBITMQ_MANIFEST_BIT=$(BITS_DIR)/rabbitmq/rabbitmq-zfs-$(_rabbitmq_stamp).imgmanifest
 
@@ -476,7 +476,7 @@ clean_rabbitmq:
 #---- DHCPD
 
 _dhcpd_stamp=$(DHCPD_BRANCH)-$(TIMESTAMP)-g$(DHCPD_SHA)
-DHCPD_BITS=$(BITS_DIR)/dhcpd/dhcpd-pkg-$(_dhcpd_stamp).tar.bz2
+DHCPD_BITS=$(BITS_DIR)/dhcpd/dhcpd-pkg-$(_dhcpd_stamp).tar.gz
 DHCPD_IMAGE_BIT=$(BITS_DIR)/dhcpd/dhcpd-zfs-$(_dhcpd_stamp).zfs.gz
 DHCPD_MANIFEST_BIT=$(BITS_DIR)/dhcpd/dhcpd-zfs-$(_dhcpd_stamp).imgmanifest
 
@@ -516,7 +516,7 @@ clean_dhcpd:
 #---- MOCKCLOUD
 
 _mockcloud_stamp=$(MOCKCLOUD_BRANCH)-$(TIMESTAMP)-g$(MOCKCLOUD_SHA)
-MOCKCLOUD_BITS=$(BITS_DIR)/mockcloud/mockcloud-pkg-$(_mockcloud_stamp).tar.bz2
+MOCKCLOUD_BITS=$(BITS_DIR)/mockcloud/mockcloud-pkg-$(_mockcloud_stamp).tar.gz
 MOCKCLOUD_IMAGE_BIT=$(BITS_DIR)/mockcloud/mockcloud-zfs-$(_mockcloud_stamp).zfs.gz
 MOCKCLOUD_MANIFEST_BIT=$(BITS_DIR)/mockcloud/mockcloud-zfs-$(_mockcloud_stamp).imgmanifest
 
@@ -557,7 +557,7 @@ clean_mockcloud:
 #---- CLOUDAPI
 
 _cloudapi_stamp=$(SDC_CLOUDAPI_BRANCH)-$(TIMESTAMP)-g$(SDC_CLOUDAPI_SHA)
-CLOUDAPI_BITS=$(BITS_DIR)/cloudapi/cloudapi-pkg-$(_cloudapi_stamp).tar.bz2
+CLOUDAPI_BITS=$(BITS_DIR)/cloudapi/cloudapi-pkg-$(_cloudapi_stamp).tar.gz
 CLOUDAPI_IMAGE_BIT=$(BITS_DIR)/cloudapi/cloudapi-zfs-$(_cloudapi_stamp).zfs.gz
 CLOUDAPI_MANIFEST_BIT=$(BITS_DIR)/cloudapi/cloudapi-zfs-$(_cloudapi_stamp).imgmanifest
 
@@ -602,7 +602,7 @@ clean_cloudapi:
 #---- NAT
 
 _nat_stamp=$(SDC_NAT_BRANCH)-$(TIMESTAMP)-g$(SDC_NAT_SHA)
-NAT_BITS=$(BITS_DIR)/nat/nat-pkg-$(_nat_stamp).tar.bz2
+NAT_BITS=$(BITS_DIR)/nat/nat-pkg-$(_nat_stamp).tar.gz
 NAT_IMAGE_BIT=$(BITS_DIR)/nat/nat-zfs-$(_nat_stamp).zfs.gz
 NAT_MANIFEST_BIT=$(BITS_DIR)/nat/nat-zfs-$(_nat_stamp).imgmanifest
 
@@ -642,7 +642,7 @@ clean_nat:
 #---- DOCKER
 
 _docker_stamp=$(SDC_DOCKER_BRANCH)-$(TIMESTAMP)-g$(SDC_DOCKER_SHA)
-DOCKER_BITS=$(BITS_DIR)/docker/docker-pkg-$(_docker_stamp).tar.bz2
+DOCKER_BITS=$(BITS_DIR)/docker/docker-pkg-$(_docker_stamp).tar.gz
 DOCKER_IMAGE_BIT=$(BITS_DIR)/docker/docker-zfs-$(_docker_stamp).zfs.gz
 DOCKER_MANIFEST_BIT=$(BITS_DIR)/docker/docker-zfs-$(_docker_stamp).imgmanifest
 
@@ -682,7 +682,7 @@ clean_docker:
 #---- PORTOLAN
 
 _portolan_stamp=$(SDC_PORTOLAN_BRANCH)-$(TIMESTAMP)-g$(SDC_PORTOLAN_SHA)
-PORTOLAN_BITS=$(BITS_DIR)/portolan/portolan-pkg-$(_portolan_stamp).tar.bz2
+PORTOLAN_BITS=$(BITS_DIR)/portolan/portolan-pkg-$(_portolan_stamp).tar.gz
 PORTOLAN_IMAGE_BIT=$(BITS_DIR)/portolan/portolan-zfs-$(_portolan_stamp).zfs.gz
 PORTOLAN_MANIFEST_BIT=$(BITS_DIR)/portolan/portolan-zfs-$(_portolan_stamp).imgmanifest
 
@@ -721,7 +721,7 @@ clean_portolan:
 #---- MANTA_MANATEE
 
 _manta-manatee_stamp=$(MANTA_MANATEE_BRANCH)-$(TIMESTAMP)-g$(MANTA_MANATEE_SHA)
-MANTA_MANATEE_BITS=$(BITS_DIR)/manta-manatee/manta-manatee-pkg-$(_manta-manatee_stamp).tar.bz2
+MANTA_MANATEE_BITS=$(BITS_DIR)/manta-manatee/manta-manatee-pkg-$(_manta-manatee_stamp).tar.gz
 MANTA_MANATEE_IMAGE_BIT=$(BITS_DIR)/manta-manatee/manta-manatee-zfs-$(_manta-manatee_stamp).zfs.gz
 MANTA_MANATEE_MANIFEST_BIT=$(BITS_DIR)/manta-manatee/manta-manatee-zfs-$(_manta-manatee_stamp).imgmanifest
 
@@ -762,7 +762,7 @@ clean_manta-manatee:
 #---- SDC_MANATEE
 
 _sdc-manatee_stamp=$(SDC_MANATEE_BRANCH)-$(TIMESTAMP)-g$(SDC_MANATEE_SHA)
-SDC_MANATEE_BITS=$(BITS_DIR)/sdc-manatee/sdc-manatee-pkg-$(_sdc-manatee_stamp).tar.bz2
+SDC_MANATEE_BITS=$(BITS_DIR)/sdc-manatee/sdc-manatee-pkg-$(_sdc-manatee_stamp).tar.gz
 SDC_MANATEE_IMAGE_BIT=$(BITS_DIR)/sdc-manatee/sdc-manatee-zfs-$(_sdc-manatee_stamp).zfs.gz
 SDC_MANATEE_MANIFEST_BIT=$(BITS_DIR)/sdc-manatee/sdc-manatee-zfs-$(_sdc-manatee_stamp).imgmanifest
 
@@ -803,7 +803,7 @@ clean_sdc-manatee:
 #---- MANATEE
 
 _manatee_stamp=$(MANATEE_BRANCH)-$(TIMESTAMP)-g$(MANATEE_SHA)
-MANATEE_BITS=$(BITS_DIR)/manatee/manatee-pkg-$(_manatee_stamp).tar.bz2
+MANATEE_BITS=$(BITS_DIR)/manatee/manatee-pkg-$(_manatee_stamp).tar.gz
 MANATEE_IMAGE_BIT=$(BITS_DIR)/manatee/manatee-zfs-$(_manatee_stamp).zfs.gz
 MANATEE_MANIFEST_BIT=$(BITS_DIR)/manatee/manatee-zfs-$(_manatee_stamp).imgmanifest
 
@@ -844,7 +844,7 @@ clean_manatee:
 #---- WORKFLOW
 
 _wf_stamp=$(SDC_WORKFLOW_BRANCH)-$(TIMESTAMP)-g$(SDC_WORKFLOW_SHA)
-WORKFLOW_BITS=$(BITS_DIR)/workflow/workflow-pkg-$(_wf_stamp).tar.bz2
+WORKFLOW_BITS=$(BITS_DIR)/workflow/workflow-pkg-$(_wf_stamp).tar.gz
 WORKFLOW_IMAGE_BIT=$(BITS_DIR)/workflow/workflow-zfs-$(_wf_stamp).zfs.gz
 WORKFLOW_MANIFEST_BIT=$(BITS_DIR)/workflow/workflow-zfs-$(_wf_stamp).imgmanifest
 
@@ -888,7 +888,7 @@ clean_workflow:
 #---- CMON
 
 _cmon_stamp=$(TRITON_CMON_BRANCH)-$(TIMESTAMP)-g$(TRITON_CMON_SHA)
-CMON_BITS=$(BITS_DIR)/cmon/cmon-pkg-$(_cmon_stamp).tar.bz2
+CMON_BITS=$(BITS_DIR)/cmon/cmon-pkg-$(_cmon_stamp).tar.gz
 CMON_IMAGE_BIT=$(BITS_DIR)/cmon/cmon-zfs-$(_cmon_stamp).zfs.gz
 CMON_MANIFEST_BIT=$(BITS_DIR)/cmon/cmon-zfs-$(_cmon_stamp).imgmanifest
 
@@ -954,7 +954,7 @@ clean_cmon_agent:
 #---- VMAPI
 
 _vmapi_stamp=$(SDC_VMAPI_BRANCH)-$(TIMESTAMP)-g$(SDC_VMAPI_SHA)
-VMAPI_BITS=$(BITS_DIR)/vmapi/vmapi-pkg-$(_vmapi_stamp).tar.bz2
+VMAPI_BITS=$(BITS_DIR)/vmapi/vmapi-pkg-$(_vmapi_stamp).tar.gz
 VMAPI_IMAGE_BIT=$(BITS_DIR)/vmapi/vmapi-zfs-$(_vmapi_stamp).zfs.gz
 VMAPI_MANIFEST_BIT=$(BITS_DIR)/vmapi/vmapi-zfs-$(_vmapi_stamp).imgmanifest
 
@@ -999,7 +999,7 @@ clean_vmapi:
 #---- PAPI
 
 _papi_stamp=$(SDC_PAPI_BRANCH)-$(TIMESTAMP)-g$(SDC_PAPI_SHA)
-PAPI_BITS=$(BITS_DIR)/papi/papi-pkg-$(_papi_stamp).tar.bz2
+PAPI_BITS=$(BITS_DIR)/papi/papi-pkg-$(_papi_stamp).tar.gz
 PAPI_IMAGE_BIT=$(BITS_DIR)/papi/papi-zfs-$(_papi_stamp).zfs.gz
 PAPI_MANIFEST_BIT=$(BITS_DIR)/papi/papi-zfs-$(_papi_stamp).imgmanifest
 
@@ -1045,7 +1045,7 @@ clean_papi:
 #---- IMGAPI
 
 _imgapi_stamp=$(SDC_IMGAPI_BRANCH)-$(TIMESTAMP)-g$(SDC_IMGAPI_SHA)
-IMGAPI_BITS=$(BITS_DIR)/imgapi/imgapi-pkg-$(_imgapi_stamp).tar.bz2
+IMGAPI_BITS=$(BITS_DIR)/imgapi/imgapi-pkg-$(_imgapi_stamp).tar.gz
 IMGAPI_IMAGE_BIT=$(BITS_DIR)/imgapi/imgapi-zfs-$(_imgapi_stamp).zfs.gz
 IMGAPI_MANIFEST_BIT=$(BITS_DIR)/imgapi/imgapi-zfs-$(_imgapi_stamp).imgmanifest
 
@@ -1085,7 +1085,7 @@ clean_imgapi:
 #---- sdc
 
 _sdc_stamp=$(SDC_SDC_BRANCH)-$(TIMESTAMP)-g$(SDC_SDC_SHA)
-SDC_BITS=$(BITS_DIR)/sdc/sdc-pkg-$(_sdc_stamp).tar.bz2
+SDC_BITS=$(BITS_DIR)/sdc/sdc-pkg-$(_sdc_stamp).tar.gz
 SDC_IMAGE_BIT=$(BITS_DIR)/sdc/sdc-zfs-$(_sdc_stamp).zfs.gz
 SDC_MANIFEST_BIT=$(BITS_DIR)/sdc/sdc-zfs-$(_sdc_stamp).imgmanifest
 
@@ -1285,7 +1285,7 @@ clean_cn_agent:
 #---- Configuration Agent
 
 _config_agent_stamp=$(SDC_CONFIG_AGENT_BRANCH)-$(TIMESTAMP)-g$(SDC_CONFIG_AGENT_SHA)
-CONFIG_AGENT_BIT=$(BITS_DIR)/config-agent/config-agent-pkg-$(_config_agent_stamp).tar.bz2
+CONFIG_AGENT_BIT=$(BITS_DIR)/config-agent/config-agent-pkg-$(_config_agent_stamp).tar.gz
 CONFIG_AGENT_MANIFEST_BIT=$(BITS_DIR)/config-agent/config-agent-pkg-$(_config_agent_stamp).manifest
 
 .PHONY: config-agent
@@ -1364,7 +1364,7 @@ clean_firewaller:
 #---- CNAPI
 
 _cnapi_stamp=$(SDC_CNAPI_BRANCH)-$(TIMESTAMP)-g$(SDC_CNAPI_SHA)
-CNAPI_BITS=$(BITS_DIR)/cnapi/cnapi-pkg-$(_cnapi_stamp).tar.bz2
+CNAPI_BITS=$(BITS_DIR)/cnapi/cnapi-pkg-$(_cnapi_stamp).tar.gz
 CNAPI_IMAGE_BIT=$(BITS_DIR)/cnapi/cnapi-zfs-$(_cnapi_stamp).zfs.gz
 CNAPI_MANIFEST_BIT=$(BITS_DIR)/cnapi/cnapi-zfs-$(_cnapi_stamp).imgmanifest
 
@@ -1408,7 +1408,7 @@ clean_cnapi:
 #---- FWAPI
 
 _fwapi_stamp=$(SDC_FWAPI_BRANCH)-$(TIMESTAMP)-g$(SDC_FWAPI_SHA)
-FWAPI_BITS=$(BITS_DIR)/fwapi/fwapi-pkg-$(_fwapi_stamp).tar.bz2
+FWAPI_BITS=$(BITS_DIR)/fwapi/fwapi-pkg-$(_fwapi_stamp).tar.gz
 FWAPI_IMAGE_BIT=$(BITS_DIR)/fwapi/fwapi-zfs-$(_fwapi_stamp).zfs.gz
 FWAPI_MANIFEST_BIT=$(BITS_DIR)/fwapi/fwapi-zfs-$(_fwapi_stamp).imgmanifest
 
@@ -1453,7 +1453,7 @@ clean_fwapi:
 #---- NAPI
 
 _napi_stamp=$(SDC_NAPI_BRANCH)-$(TIMESTAMP)-g$(SDC_NAPI_SHA)
-NAPI_BITS=$(BITS_DIR)/napi/napi-pkg-$(_napi_stamp).tar.bz2
+NAPI_BITS=$(BITS_DIR)/napi/napi-pkg-$(_napi_stamp).tar.gz
 NAPI_IMAGE_BIT=$(BITS_DIR)/napi/napi-zfs-$(_napi_stamp).zfs.gz
 NAPI_MANIFEST_BIT=$(BITS_DIR)/napi/napi-zfs-$(_napi_stamp).imgmanifest
 
@@ -1498,7 +1498,7 @@ clean_napi:
 #---- SAPI
 
 _sapi_stamp=$(SDC_SAPI_BRANCH)-$(TIMESTAMP)-g$(SDC_SAPI_SHA)
-SAPI_BITS=$(BITS_DIR)/sapi/sapi-pkg-$(_sapi_stamp).tar.bz2
+SAPI_BITS=$(BITS_DIR)/sapi/sapi-pkg-$(_sapi_stamp).tar.gz
 SAPI_IMAGE_BIT=$(BITS_DIR)/sapi/sapi-zfs-$(_sapi_stamp).zfs.gz
 SAPI_MANIFEST_BIT=$(BITS_DIR)/sapi/sapi-zfs-$(_sapi_stamp).imgmanifest
 
@@ -1543,7 +1543,7 @@ clean_sapi:
 #---- SDCSSO
 
 _sdcsso_stamp=$(SDCSSO_BRANCH)-$(TIMESTAMP)-g$(SDCSSO_SHA)
-SDCSSO_BITS=$(BITS_DIR)/sdcsso/sdcsso-pkg-$(_sdcsso_stamp).tar.bz2
+SDCSSO_BITS=$(BITS_DIR)/sdcsso/sdcsso-pkg-$(_sdcsso_stamp).tar.gz
 SDCSSO_IMAGE_BIT=$(BITS_DIR)/sdcsso/sdcsso-zfs-$(_sdcsso_stamp).zfs.gz
 SDCSSO_MANIFEST_BIT=$(BITS_DIR)/sdcsso/sdcsso-zfs-$(_sdcsso_stamp).imgmanifest
 
@@ -1585,7 +1585,7 @@ clean_sdcsso:
 #---- Marlin
 
 _marlin_stamp=$(MANTA_MARLIN_BRANCH)-$(TIMESTAMP)-g$(MANTA_MARLIN_SHA)
-MARLIN_BITS=$(BITS_DIR)/marlin/marlin-pkg-$(_marlin_stamp).tar.bz2
+MARLIN_BITS=$(BITS_DIR)/marlin/marlin-pkg-$(_marlin_stamp).tar.gz
 MARLIN_IMAGE_BIT=$(BITS_DIR)/marlin/marlin-zfs-$(_marlin_stamp).zfs.gz
 MARLIN_MANIFEST_BIT=$(BITS_DIR)/marlin/marlin-zfs-$(_marlin_stamp).imgmanifest
 MARLIN_AGENT_BIT=$(BITS_DIR)/marlin/marlin-$(_marlin_stamp).tar.gz
@@ -1630,7 +1630,7 @@ clean_marlin:
 #---- MEDUSA
 
 _medusa_stamp=$(MANTA_MEDUSA_BRANCH)-$(TIMESTAMP)-g$(MANTA_MEDUSA_SHA)
-MEDUSA_BITS=$(BITS_DIR)/medusa/medusa-pkg-$(_medusa_stamp).tar.bz2
+MEDUSA_BITS=$(BITS_DIR)/medusa/medusa-pkg-$(_medusa_stamp).tar.gz
 MEDUSA_IMAGE_BIT=$(BITS_DIR)/medusa/medusa-zfs-$(_medusa_stamp).zfs.gz
 MEDUSA_MANIFEST_BIT=$(BITS_DIR)/medusa/medusa-zfs-$(_medusa_stamp).imgmanifest
 
@@ -1672,7 +1672,7 @@ clean_medusa:
 #---- MAHI
 
 _mahi_stamp=$(MAHI_BRANCH)-$(TIMESTAMP)-g$(MAHI_SHA)
-MAHI_BITS=$(BITS_DIR)/mahi/mahi-pkg-$(_mahi_stamp).tar.bz2
+MAHI_BITS=$(BITS_DIR)/mahi/mahi-pkg-$(_mahi_stamp).tar.gz
 MAHI_IMAGE_BIT=$(BITS_DIR)/mahi/mahi-zfs-$(_mahi_stamp).zfs.gz
 MAHI_MANIFEST_BIT=$(BITS_DIR)/mahi/mahi-zfs-$(_mahi_stamp).imgmanifest
 
@@ -1712,7 +1712,7 @@ clean_mahi:
 #---- Triton CNS
 
 _cns_stamp=$(TRITON_CNS_BRANCH)-$(TIMESTAMP)-g$(TRITON_CNS_SHA)
-CNS_BITS=$(BITS_DIR)/cns/cns-pkg-$(_cns_stamp).tar.bz2
+CNS_BITS=$(BITS_DIR)/cns/cns-pkg-$(_cns_stamp).tar.gz
 CNS_IMAGE_BIT=$(BITS_DIR)/cns/cns-zfs-$(_cns_stamp).zfs.gz
 CNS_MANIFEST_BIT=$(BITS_DIR)/cns/cns-zfs-$(_cns_stamp).imgmanifest
 
@@ -1753,7 +1753,7 @@ clean_cns:
 #---- Manta Resharding System
 
 _manta-reshard_stamp=$(MANTA_RESHARD_BRANCH)-$(TIMESTAMP)-g$(MANTA_RESHARD_SHA)
-MANTA_RESHARD_BITS=$(BITS_DIR)/manta-reshard/manta-reshard-pkg-$(_manta-reshard_stamp).tar.bz2
+MANTA_RESHARD_BITS=$(BITS_DIR)/manta-reshard/manta-reshard-pkg-$(_manta-reshard_stamp).tar.gz
 MANTA_RESHARD_IMAGE_BIT=$(BITS_DIR)/manta-reshard/manta-reshard-zfs-$(_manta-reshard_stamp).zfs.gz
 MANTA_RESHARD_MANIFEST_BIT=$(BITS_DIR)/manta-reshard/manta-reshard-zfs-$(_manta-reshard_stamp).imgmanifest
 
@@ -1794,7 +1794,7 @@ clean_manta-reshard:
 #---- pgstatsmon
 
 _pgstatsmon_stamp=$(PGSTATSMON_BRANCH)-$(TIMESTAMP)-g$(PGSTATSMON_SHA)
-PGSTATSMON_BITS=$(BITS_DIR)/pgstatsmon/pgstatsmon-pkg-$(_pgstatsmon_stamp).tar.bz2
+PGSTATSMON_BITS=$(BITS_DIR)/pgstatsmon/pgstatsmon-pkg-$(_pgstatsmon_stamp).tar.gz
 PGSTATSMON_IMAGE_BIT=$(BITS_DIR)/pgstatsmon/pgstatsmon-zfs-$(_pgstatsmon_stamp).zfs.gz
 PGSTATSMON_MANIFEST_BIT=$(BITS_DIR)/pgstatsmon/pgstatsmon-zfs-$(_pgstatsmon_stamp).imgmanifest
 
@@ -1835,7 +1835,7 @@ clean_pgstatsmon:
 #---- Manta Garbage Collector
 
 _manta-garbage-collector_stamp=$(MANTA_GARBAGE_COLLECTOR_BRANCH)-$(TIMESTAMP)-g$(MANTA_GARBAGE_COLLECTOR_SHA)
-MANTA_GARBAGE_COLLECTOR_BITS=$(BITS_DIR)/manta-garbage-collector/manta-garbage-collector-pkg-$(_manta-garbage-collector_stamp).tar.bz2
+MANTA_GARBAGE_COLLECTOR_BITS=$(BITS_DIR)/manta-garbage-collector/manta-garbage-collector-pkg-$(_manta-garbage-collector_stamp).tar.gz
 MANTA_GARBAGE_COLLECTOR_IMAGE_BIT=$(BITS_DIR)/manta-garbage-collector/manta-garbage-collector-zfs-$(_manta-garbage-collector_stamp).zfs.gz
 MANTA_GARBAGE_COLLECTOR_MANIFEST_BIT=$(BITS_DIR)/manta-garbage-collector/manta-garbage-collector-zfs-$(_manta-garbage-collector_stamp).imgmanifest
 
@@ -1876,7 +1876,7 @@ clean_manta-garbage-collector:
 #---- Mola
 
 _mola_stamp=$(MANTA_MOLA_BRANCH)-$(TIMESTAMP)-g$(MANTA_MOLA_SHA)
-MOLA_BITS=$(BITS_DIR)/mola/mola-pkg-$(_mola_stamp).tar.bz2
+MOLA_BITS=$(BITS_DIR)/mola/mola-pkg-$(_mola_stamp).tar.gz
 MOLA_IMAGE_BIT=$(BITS_DIR)/mola/mola-zfs-$(_mola_stamp).zfs.gz
 MOLA_MANIFEST_BIT=$(BITS_DIR)/mola/mola-zfs-$(_mola_stamp).imgmanifest
 
@@ -1919,7 +1919,7 @@ clean_mola:
 #---- Madtom
 
 _madtom_stamp=$(MANTA_MADTOM_BRANCH)-$(TIMESTAMP)-g$(MANTA_MADTOM_SHA)
-MADTOM_BITS=$(BITS_DIR)/madtom/madtom-pkg-$(_madtom_stamp).tar.bz2
+MADTOM_BITS=$(BITS_DIR)/madtom/madtom-pkg-$(_madtom_stamp).tar.gz
 MADTOM_IMAGE_BIT=$(BITS_DIR)/madtom/madtom-zfs-$(_madtom_stamp).zfs.gz
 MADTOM_MANIFEST_BIT=$(BITS_DIR)/madtom/madtom-zfs-$(_madtom_stamp).imgmanifest
 
@@ -1962,7 +1962,7 @@ clean_madtom:
 #---- Marlin Dashboard
 
 _marlin-dashboard_stamp=$(MANTA_MARLIN_DASHBOARD_BRANCH)-$(TIMESTAMP)-g$(MANTA_MARLIN_DASHBOARD_SHA)
-MARLIN_DASHBOARD_BITS=$(BITS_DIR)/marlin-dashboard/marlin-dashboard-pkg-$(_marlin-dashboard_stamp).tar.bz2
+MARLIN_DASHBOARD_BITS=$(BITS_DIR)/marlin-dashboard/marlin-dashboard-pkg-$(_marlin-dashboard_stamp).tar.gz
 MARLIN_DASHBOARD_IMAGE_BIT=$(BITS_DIR)/marlin-dashboard/marlin-dashboard-zfs-$(_marlin-dashboard_stamp).zfs.gz
 MARLIN_DASHBOARD_MANIFEST_BIT=$(BITS_DIR)/marlin-dashboard/marlin-dashboard-zfs-$(_marlin-dashboard_stamp).imgmanifest
 
@@ -2043,7 +2043,7 @@ clean_nfsserver:
 #---- VOLAPI
 
 _volapi_stamp=$(SDC_VOLAPI_BRANCH)-$(TIMESTAMP)-g$(SDC_VOLAPI_SHA)
-VOLAPI_BITS=$(BITS_DIR)/volapi/volapi-pkg-$(_volapi_stamp).tar.bz2
+VOLAPI_BITS=$(BITS_DIR)/volapi/volapi-pkg-$(_volapi_stamp).tar.gz
 VOLAPI_IMAGE_BIT=$(BITS_DIR)/volapi/volapi-zfs-$(_volapi_stamp).zfs.gz
 VOLAPI_MANIFEST_BIT=$(BITS_DIR)/volapi/volapi-zfs-$(_volapi_stamp).imgmanifest
 
@@ -2087,7 +2087,7 @@ clean_volapi:
 #---- Propeller
 
 _propeller_stamp=$(MANTA_PROPELLER_BRANCH)-$(TIMESTAMP)-g$(MANTA_PROPELLER_SHA)
-PROPELLER_BITS=$(BITS_DIR)/propeller/propeller-pkg-$(_propeller_stamp).tar.bz2
+PROPELLER_BITS=$(BITS_DIR)/propeller/propeller-pkg-$(_propeller_stamp).tar.gz
 PROPELLER_IMAGE_BIT=$(BITS_DIR)/propeller/propeller-zfs-$(_propeller_stamp).zfs.gz
 PROPELLER_MANIFEST_BIT=$(BITS_DIR)/propeller/propeller-zfs-$(_propeller_stamp).imgmanifest
 
@@ -2130,7 +2130,7 @@ clean_propeller:
 #---- Moray
 
 _moray_stamp=$(MORAY_BRANCH)-$(TIMESTAMP)-g$(MORAY_SHA)
-MORAY_BITS=$(BITS_DIR)/moray/moray-pkg-$(_moray_stamp).tar.bz2
+MORAY_BITS=$(BITS_DIR)/moray/moray-pkg-$(_moray_stamp).tar.gz
 MORAY_IMAGE_BIT=$(BITS_DIR)/moray/moray-zfs-$(_moray_stamp).zfs.gz
 MORAY_MANIFEST_BIT=$(BITS_DIR)/moray/moray-zfs-$(_moray_stamp).imgmanifest
 
@@ -2173,7 +2173,7 @@ clean_moray:
 #---- Electric-Moray
 
 _electric-moray_stamp=$(ELECTRIC_MORAY_BRANCH)-$(TIMESTAMP)-g$(ELECTRIC_MORAY_SHA)
-ELECTRIC_MORAY_BITS=$(BITS_DIR)/electric-moray/electric-moray-pkg-$(_electric-moray_stamp).tar.bz2
+ELECTRIC_MORAY_BITS=$(BITS_DIR)/electric-moray/electric-moray-pkg-$(_electric-moray_stamp).tar.gz
 ELECTRIC_MORAY_IMAGE_BIT=$(BITS_DIR)/electric-moray/electric-moray-zfs-$(_electric-moray_stamp).zfs.gz
 ELECTRIC_MORAY_MANIFEST_BIT=$(BITS_DIR)/electric-moray/electric-moray-zfs-$(_electric-moray_stamp).imgmanifest
 
@@ -2216,7 +2216,7 @@ clean_electric-moray:
 #---- Muskie
 
 _muskie_stamp=$(MANTA_MUSKIE_BRANCH)-$(TIMESTAMP)-g$(MANTA_MUSKIE_SHA)
-MUSKIE_BITS=$(BITS_DIR)/muskie/muskie-pkg-$(_muskie_stamp).tar.bz2
+MUSKIE_BITS=$(BITS_DIR)/muskie/muskie-pkg-$(_muskie_stamp).tar.gz
 MUSKIE_IMAGE_BIT=$(BITS_DIR)/muskie/muskie-zfs-$(_muskie_stamp).zfs.gz
 MUSKIE_MANIFEST_BIT=$(BITS_DIR)/muskie/muskie-zfs-$(_muskie_stamp).imgmanifest
 
@@ -2259,7 +2259,7 @@ clean_muskie:
 #---- Wrasse
 
 _wrasse_stamp=$(MANTA_WRASSE_BRANCH)-$(TIMESTAMP)-g$(MANTA_WRASSE_SHA)
-WRASSE_BITS=$(BITS_DIR)/wrasse/wrasse-pkg-$(_wrasse_stamp).tar.bz2
+WRASSE_BITS=$(BITS_DIR)/wrasse/wrasse-pkg-$(_wrasse_stamp).tar.gz
 WRASSE_IMAGE_BIT=$(BITS_DIR)/wrasse/wrasse-zfs-$(_wrasse_stamp).zfs.gz
 WRASSE_MANIFEST_BIT=$(BITS_DIR)/wrasse/wrasse-zfs-$(_wrasse_stamp).imgmanifest
 
@@ -2302,7 +2302,7 @@ clean_wrasse:
 #---- Registrar
 
 _registrar_stamp=$(REGISTRAR_BRANCH)-$(TIMESTAMP)-g$(REGISTRAR_SHA)
-REGISTRAR_BITS=$(BITS_DIR)/registrar/registrar-pkg-$(_registrar_stamp).tar.bz2
+REGISTRAR_BITS=$(BITS_DIR)/registrar/registrar-pkg-$(_registrar_stamp).tar.gz
 
 .PHONY: registrar
 registrar: $(REGISTRAR_BITS)
@@ -2323,7 +2323,7 @@ clean_registrar:
 #---- waferlock
 
 _waferlock_stamp=$(WAFERLOCK_BRANCH)-$(TIMESTAMP)-g$(WAFERLOCK_SHA)
-WAFERLOCK_BITS=$(BITS_DIR)/waferlock/waferlock-pkg-$(_waferlock_stamp).tar.bz2
+WAFERLOCK_BITS=$(BITS_DIR)/waferlock/waferlock-pkg-$(_waferlock_stamp).tar.gz
 
 .PHONY: waferlock
 waferlock: $(WAFERLOCK_BITS)
@@ -2343,7 +2343,7 @@ clean_waferlock:
 #---- mackerel
 
 _mackerel_stamp=$(MANTA_MACKEREL_BRANCH)-$(TIMESTAMP)-g$(MANTA_MACKEREL_SHA)
-MACKEREL_BITS=$(BITS_DIR)/mackerel/mackerel-pkg-$(_mackerel_stamp).tar.bz2
+MACKEREL_BITS=$(BITS_DIR)/mackerel/mackerel-pkg-$(_mackerel_stamp).tar.gz
 
 .PHONY: mackerel
 mackerel: $(MACKEREL_BITS)
@@ -2364,7 +2364,7 @@ clean_mackerel:
 #---- Binder
 
 _binder_stamp=$(BINDER_BRANCH)-$(TIMESTAMP)-g$(BINDER_SHA)
-BINDER_BITS=$(BITS_DIR)/binder/binder-pkg-$(_binder_stamp).tar.bz2
+BINDER_BITS=$(BITS_DIR)/binder/binder-pkg-$(_binder_stamp).tar.gz
 BINDER_IMAGE_BIT=$(BITS_DIR)/binder/binder-zfs-$(_binder_stamp).zfs.gz
 BINDER_MANIFEST_BIT=$(BITS_DIR)/binder/binder-zfs-$(_binder_stamp).imgmanifest
 
@@ -2405,7 +2405,7 @@ clean_binder:
 #---- Muppet
 
 _muppet_stamp=$(MUPPET_BRANCH)-$(TIMESTAMP)-g$(MUPPET_SHA)
-MUPPET_BITS=$(BITS_DIR)/muppet/muppet-pkg-$(_muppet_stamp).tar.bz2
+MUPPET_BITS=$(BITS_DIR)/muppet/muppet-pkg-$(_muppet_stamp).tar.gz
 MUPPET_IMAGE_BIT=$(BITS_DIR)/muppet/muppet-zfs-$(_muppet_stamp).zfs.gz
 MUPPET_MANIFEST_BIT=$(BITS_DIR)/muppet/muppet-zfs-$(_muppet_stamp).imgmanifest
 
@@ -2445,7 +2445,7 @@ clean_muppet:
 #---- Minnow
 
 _minnow_stamp=$(MANTA_MINNOW_BRANCH)-$(TIMESTAMP)-g$(MANTA_MINNOW_SHA)
-MINNOW_BITS=$(BITS_DIR)/minnow/minnow-pkg-$(_minnow_stamp).tar.bz2
+MINNOW_BITS=$(BITS_DIR)/minnow/minnow-pkg-$(_minnow_stamp).tar.gz
 
 .PHONY: minnow
 minnow: $(MINNOW_BITS)
@@ -2466,7 +2466,7 @@ clean_minnow:
 #---- Mako
 
 _mako_stamp=$(MANTA_MAKO_BRANCH)-$(TIMESTAMP)-g$(MANTA_MAKO_SHA)
-MAKO_BITS=$(BITS_DIR)/mako/mako-pkg-$(_mako_stamp).tar.bz2
+MAKO_BITS=$(BITS_DIR)/mako/mako-pkg-$(_mako_stamp).tar.gz
 MAKO_IMAGE_BIT=$(BITS_DIR)/mako/mako-zfs-$(_mako_stamp).zfs.gz
 MAKO_MANIFEST_BIT=$(BITS_DIR)/mako/mako-zfs-$(_mako_stamp).imgmanifest
 
@@ -2599,7 +2599,7 @@ clean_dockerlogger:
 #---- Manta deployment (the manta zone)
 
 _manta_deployment_stamp=$(SDC_MANTA_BRANCH)-$(TIMESTAMP)-g$(SDC_MANTA_SHA)
-MANTA_DEPLOYMENT_BITS=$(BITS_DIR)/manta-deployment/manta-deployment-pkg-$(_manta_deployment_stamp).tar.bz2
+MANTA_DEPLOYMENT_BITS=$(BITS_DIR)/manta-deployment/manta-deployment-pkg-$(_manta_deployment_stamp).tar.gz
 MANTA_DEPLOYMENT_IMAGE_BIT=$(BITS_DIR)/manta-deployment/manta-deployment-zfs-$(_manta_deployment_stamp).zfs.gz
 MANTA_DEPLOYMENT_MANIFEST_BIT=$(BITS_DIR)/manta-deployment/manta-deployment-zfs-$(_manta_deployment_stamp).imgmanifest
 
diff --git a/targets.json.in b/targets.json.in
index 2e41d6d..880ad0c 100644
--- a/targets.json.in
+++ b/targets.json.in
@@ -125,9 +125,9 @@ cat <<EOF
       "https://download.joyent.com/pub/build/sdcnode"
     ],
     "tarballs": [
-      {"tarball": "registrar/registrar-pkg-*.tar.bz2", "sysroot": "/"},
+      {"tarball": "registrar/registrar-pkg-*.tar.gz", "sysroot": "/"},
       {"tarball": "amon/amon-agent-*.tgz", "sysroot": "/opt"},
-      {"tarball": "config-agent/config-agent-*.tar.bz2", "sysroot": "/opt/smartdc"}
+      {"tarball": "config-agent/config-agent-*.tar.gz", "sysroot": "/opt/smartdc"}
     ]
   },
   "amon": {
@@ -148,9 +148,9 @@ cat <<EOF
       "https://download.joyent.com/pub/build/sdcnode"
     ],
     "tarballs": [
-      {"tarball": "registrar/registrar-pkg-*.tar.bz2", "sysroot": "/"},
+      {"tarball": "registrar/registrar-pkg-*.tar.gz", "sysroot": "/"},
       {"tarball": "amon/amon-agent-*.tgz", "sysroot": "/opt"},
-      {"tarball": "config-agent/config-agent-*.tar.bz2", "sysroot": "/opt/smartdc"}
+      {"tarball": "config-agent/config-agent-*.tar.gz", "sysroot": "/opt/smartdc"}
     ]
   },
   "assets": {
@@ -196,9 +196,9 @@ cat <<EOF
       "https://download.joyent.com/pub/build/sdcnode"
     ],
     "tarballs": [
-      {"tarball": "registrar/registrar-pkg-*.tar.bz2", "sysroot": "/"},
+      {"tarball": "registrar/registrar-pkg-*.tar.gz", "sysroot": "/"},
       {"tarball": "amon/amon-agent-*.tgz", "sysroot": "/opt"},
-      {"tarball": "config-agent/config-agent-*.tar.bz2", "sysroot": "/opt/smartdc"}
+      {"tarball": "config-agent/config-agent-*.tar.gz", "sysroot": "/opt/smartdc"}
     ]
   },
   "adminui": {
@@ -221,9 +221,9 @@ cat <<EOF
       "https://download.joyent.com/pub/build/sdcnode"
     ],
     "tarballs": [
-      {"tarball": "registrar/registrar-pkg-*.tar.bz2", "sysroot": "/"},
+      {"tarball": "registrar/registrar-pkg-*.tar.gz", "sysroot": "/"},
       {"tarball": "amon/amon-agent-*.tgz", "sysroot": "/opt"},
-      {"tarball": "config-agent/config-agent-*.tar.bz2", "sysroot": "/opt/smartdc"}
+      {"tarball": "config-agent/config-agent-*.tar.gz", "sysroot": "/opt/smartdc"}
     ]
   },
   "mockcloud": {
@@ -283,9 +283,9 @@ cat <<EOF
       "https://download.joyent.com/pub/build/sdcnode"
     ],
     "tarballs": [
-      {"tarball": "registrar/registrar-pkg-*.tar.bz2", "sysroot": "/"},
+      {"tarball": "registrar/registrar-pkg-*.tar.gz", "sysroot": "/"},
       {"tarball": "amon/amon-agent-*.tgz", "sysroot": "/opt"},
-      {"tarball": "config-agent/config-agent-*.tar.bz2", "sysroot": "/opt/smartdc"}
+      {"tarball": "config-agent/config-agent-*.tar.gz", "sysroot": "/opt/smartdc"}
     ]
   },
   "amonredis": {
@@ -307,9 +307,9 @@ cat <<EOF
       "registrar"
     ],
     "tarballs": [
-      {"tarball": "registrar/registrar-pkg-*.tar.bz2", "sysroot": "/"},
+      {"tarball": "registrar/registrar-pkg-*.tar.gz", "sysroot": "/"},
       {"tarball": "amon/amon-agent-*.tgz", "sysroot": "/opt"},
-      {"tarball": "config-agent/config-agent-*.tar.bz2", "sysroot": "/opt/smartdc"}
+      {"tarball": "config-agent/config-agent-*.tar.gz", "sysroot": "/opt/smartdc"}
     ]
   },
   "rabbitmq": {
@@ -339,9 +339,9 @@ cat <<EOF
       "registrar"
     ],
     "tarballs": [
-      {"tarball": "registrar/registrar-pkg-*.tar.bz2", "sysroot": "/"},
+      {"tarball": "registrar/registrar-pkg-*.tar.gz", "sysroot": "/"},
       {"tarball": "amon/amon-agent-*.tgz", "sysroot": "/opt"},
-      {"tarball": "config-agent/config-agent-*.tar.bz2", "sysroot": "/opt/smartdc"}
+      {"tarball": "config-agent/config-agent-*.tar.gz", "sysroot": "/opt/smartdc"}
     ]
   },
   "cloudapi": {
@@ -368,9 +368,9 @@ cat <<EOF
       "haproxy-1.6.2"
     ],
     "tarballs": [
-      {"tarball": "registrar/registrar-pkg-*.tar.bz2", "sysroot": "/"},
+      {"tarball": "registrar/registrar-pkg-*.tar.gz", "sysroot": "/"},
       {"tarball": "amon/amon-agent-*.tgz", "sysroot": "/opt"},
-      {"tarball": "config-agent/config-agent-*.tar.bz2", "sysroot": "/opt/smartdc"}
+      {"tarball": "config-agent/config-agent-*.tar.gz", "sysroot": "/opt/smartdc"}
     ]
   },
   "nat": {
@@ -408,9 +408,9 @@ cat <<EOF
     ],
     "pkgsrc": [],
     "tarballs": [
-      {"tarball": "registrar/registrar-pkg-*.tar.bz2", "sysroot": "/"},
+      {"tarball": "registrar/registrar-pkg-*.tar.gz", "sysroot": "/"},
       {"tarball": "amon/amon-agent-*.tgz", "sysroot": "/opt"},
-      {"tarball": "config-agent/config-agent-*.tar.bz2", "sysroot": "/opt/smartdc"}
+      {"tarball": "config-agent/config-agent-*.tar.gz", "sysroot": "/opt/smartdc"}
     ]
   },
   "volapi": {
@@ -433,9 +433,9 @@ cat <<EOF
     ],
     "pkgsrc": [],
     "tarballs": [
-      {"tarball": "registrar/registrar-pkg-*.tar.bz2", "sysroot": "/"},
+      {"tarball": "registrar/registrar-pkg-*.tar.gz", "sysroot": "/"},
       {"tarball": "amon/amon-agent-*.tgz", "sysroot": "/opt"},
-      {"tarball": "config-agent/config-agent-*.tar.bz2", "sysroot": "/opt/smartdc"}
+      {"tarball": "config-agent/config-agent-*.tar.gz", "sysroot": "/opt/smartdc"}
     ]
   },
   "portolan": {
@@ -457,9 +457,9 @@ cat <<EOF
     ],
     "pkgsrc": [],
     "tarballs": [
-      {"tarball": "registrar/registrar-pkg-*.tar.bz2", "sysroot": "/"},
+      {"tarball": "registrar/registrar-pkg-*.tar.gz", "sysroot": "/"},
       {"tarball": "amon/amon-agent-*.tgz", "sysroot": "/opt"},
-      {"tarball": "config-agent/config-agent-*.tar.bz2", "sysroot": "/opt/smartdc"}
+      {"tarball": "config-agent/config-agent-*.tar.gz", "sysroot": "/opt/smartdc"}
     ]
   },
 
@@ -485,9 +485,9 @@ cat <<EOF
       "https://download.joyent.com/pub/build/sdcnode"
     ],
     "tarballs": [
-      {"tarball": "registrar/registrar-pkg-*.tar.bz2", "sysroot": "/"},
+      {"tarball": "registrar/registrar-pkg-*.tar.gz", "sysroot": "/"},
       {"tarball": "amon/amon-agent-*.tgz", "sysroot": "/opt"},
-      {"tarball": "config-agent/config-agent-*.tar.bz2", "sysroot": "/opt/smartdc"}
+      {"tarball": "config-agent/config-agent-*.tar.gz", "sysroot": "/opt/smartdc"}
     ]
   },
 
@@ -513,9 +513,9 @@ cat <<EOF
       "https://download.joyent.com/pub/build/sdcnode"
     ],
     "tarballs": [
-      {"tarball": "registrar/registrar-pkg-*.tar.bz2", "sysroot": "/"},
+      {"tarball": "registrar/registrar-pkg-*.tar.gz", "sysroot": "/"},
       {"tarball": "amon/amon-agent-*.tgz", "sysroot": "/opt"},
-      {"tarball": "config-agent/config-agent-*.tar.bz2", "sysroot": "/opt/smartdc"}
+      {"tarball": "config-agent/config-agent-*.tar.gz", "sysroot": "/opt/smartdc"}
     ]
   },
 
@@ -537,8 +537,8 @@ cat <<EOF
       "https://download.joyent.com/pub/build/sdcnode"
     ],
     "tarballs": [
-      {"tarball": "registrar/registrar-pkg-*.tar.bz2", "sysroot": "/"},
-      {"tarball": "config-agent/config-agent-*.tar.bz2", "sysroot": "/opt/smartdc"}
+      {"tarball": "registrar/registrar-pkg-*.tar.gz", "sysroot": "/"},
+      {"tarball": "config-agent/config-agent-*.tar.gz", "sysroot": "/opt/smartdc"}
     ]
   },
 
@@ -572,9 +572,9 @@ cat <<EOF
       "https://download.joyent.com/pub/build/sdcnode"
     ],
     "tarballs": [
-      {"tarball": "registrar/registrar-pkg-*.tar.bz2", "sysroot": "/"},
+      {"tarball": "registrar/registrar-pkg-*.tar.gz", "sysroot": "/"},
       {"tarball": "amon/amon-agent-*.tgz", "sysroot": "/opt"},
-      {"tarball": "config-agent/config-agent-*.tar.bz2", "sysroot": "/opt/smartdc"}
+      {"tarball": "config-agent/config-agent-*.tar.gz", "sysroot": "/opt/smartdc"}
     ]
   },
 
@@ -598,9 +598,9 @@ cat <<EOF
       "https://download.joyent.com/pub/build/sdcnode"
     ],
     "tarballs": [
-      {"tarball": "registrar/registrar-pkg-*.tar.bz2", "sysroot": "/"},
+      {"tarball": "registrar/registrar-pkg-*.tar.gz", "sysroot": "/"},
       {"tarball": "amon/amon-agent-*.tgz", "sysroot": "/opt"},
-      {"tarball": "config-agent/config-agent-*.tar.bz2", "sysroot": "/opt/smartdc"}
+      {"tarball": "config-agent/config-agent-*.tar.gz", "sysroot": "/opt/smartdc"}
     ]
   },
 
@@ -631,9 +631,9 @@ cat <<EOF
       "https://download.joyent.com/pub/build/sdcnode"
     ],
     "tarballs": [
-      {"tarball": "registrar/registrar-pkg-*.tar.bz2", "sysroot": "/"},
+      {"tarball": "registrar/registrar-pkg-*.tar.gz", "sysroot": "/"},
       {"tarball": "amon/amon-agent-*.tgz", "sysroot": "/opt"},
-      {"tarball": "config-agent/config-agent-*.tar.bz2", "sysroot": "/opt/smartdc"}
+      {"tarball": "config-agent/config-agent-*.tar.gz", "sysroot": "/opt/smartdc"}
     ]
   },
 
@@ -658,7 +658,7 @@ cat <<EOF
     ],
     "tarballs": [
       {"tarball": "amon/amon-agent-*.tgz", "sysroot": "/opt"},
-      {"tarball": "config-agent/config-agent-*.tar.bz2", "sysroot": "/opt/smartdc"}
+      {"tarball": "config-agent/config-agent-*.tar.gz", "sysroot": "/opt/smartdc"}
     ]
   },
 
@@ -759,9 +759,9 @@ cat <<EOF
       "https://download.joyent.com/pub/build/sdcnode"
     ],
     "tarballs": [
-      {"tarball": "registrar/registrar-pkg-*.tar.bz2", "sysroot": "/"},
+      {"tarball": "registrar/registrar-pkg-*.tar.gz", "sysroot": "/"},
       {"tarball": "amon/amon-agent-*.tgz", "sysroot": "/opt"},
-      {"tarball": "config-agent/config-agent-*.tar.bz2", "sysroot": "/opt/smartdc"}
+      {"tarball": "config-agent/config-agent-*.tar.gz", "sysroot": "/opt/smartdc"}
     ]
   },
 
@@ -785,9 +785,9 @@ cat <<EOF
       "https://download.joyent.com/pub/build/sdcnode"
     ],
     "tarballs": [
-      {"tarball": "registrar/registrar-pkg-*.tar.bz2", "sysroot": "/"},
+      {"tarball": "registrar/registrar-pkg-*.tar.gz", "sysroot": "/"},
       {"tarball": "amon/amon-agent-*.tgz", "sysroot": "/opt"},
-      {"tarball": "config-agent/config-agent-*.tar.bz2", "sysroot": "/opt/smartdc"}
+      {"tarball": "config-agent/config-agent-*.tar.gz", "sysroot": "/opt/smartdc"}
     ]
   },
 
@@ -810,9 +810,9 @@ cat <<EOF
       "https://download.joyent.com/pub/build/sdcnode"
     ],
     "tarballs": [
-      {"tarball": "registrar/registrar-pkg-*.tar.bz2", "sysroot": "/"},
+      {"tarball": "registrar/registrar-pkg-*.tar.gz", "sysroot": "/"},
       {"tarball": "amon/amon-agent-*.tgz", "sysroot": "/opt"},
-      {"tarball": "config-agent/config-agent-*.tar.bz2", "sysroot": "/opt/smartdc"}
+      {"tarball": "config-agent/config-agent-*.tar.gz", "sysroot": "/opt/smartdc"}
     ]
   },
 
@@ -836,9 +836,9 @@ cat <<EOF
       "https://download.joyent.com/pub/build/sdcnode"
     ],
     "tarballs": [
-      {"tarball": "registrar/registrar-pkg-*.tar.bz2", "sysroot": "/"},
+      {"tarball": "registrar/registrar-pkg-*.tar.gz", "sysroot": "/"},
       {"tarball": "amon/amon-agent-*.tgz", "sysroot": "/opt"},
-      {"tarball": "config-agent/config-agent-*.tar.bz2", "sysroot": "/opt/smartdc"}
+      {"tarball": "config-agent/config-agent-*.tar.gz", "sysroot": "/opt/smartdc"}
     ]
   },
 
@@ -908,9 +908,9 @@ cat <<EOF
       "config-agent"
     ],
     "tarballs": [
-      {"tarball": "registrar/registrar-pkg-*.tar.bz2", "sysroot": "/"},
+      {"tarball": "registrar/registrar-pkg-*.tar.gz", "sysroot": "/"},
       {"tarball": "amon/amon-agent-*.tgz", "sysroot": "/opt"},
-      {"tarball": "config-agent/config-agent-*.tar.bz2", "sysroot": "/opt/smartdc"}
+      {"tarball": "config-agent/config-agent-*.tar.gz", "sysroot": "/opt/smartdc"}
     ]
   },
 
@@ -936,10 +936,10 @@ cat <<EOF
       "config-agent"
     ],
     "tarballs": [
-      {"tarball": "registrar/registrar-pkg-*.tar.bz2", "sysroot": "/"},
-      {"tarball": "waferlock/waferlock-pkg-*.tar.bz2", "sysroot": "/"},
+      {"tarball": "registrar/registrar-pkg-*.tar.gz", "sysroot": "/"},
+      {"tarball": "waferlock/waferlock-pkg-*.tar.gz", "sysroot": "/"},
       {"tarball": "amon/amon-agent-*.tgz", "sysroot": "/opt"},
-      {"tarball": "config-agent/config-agent-*.tar.bz2", "sysroot": "/opt/smartdc"}
+      {"tarball": "config-agent/config-agent-*.tar.gz", "sysroot": "/opt/smartdc"}
     ]
   },
 
@@ -965,10 +965,10 @@ cat <<EOF
       "config-agent"
     ],
     "tarballs": [
-      {"tarball": "registrar/registrar-pkg-*.tar.bz2", "sysroot": "/"},
-      {"tarball": "waferlock/waferlock-pkg-*.tar.bz2", "sysroot": "/"},
+      {"tarball": "registrar/registrar-pkg-*.tar.gz", "sysroot": "/"},
+      {"tarball": "waferlock/waferlock-pkg-*.tar.gz", "sysroot": "/"},
       {"tarball": "amon/amon-agent-*.tgz", "sysroot": "/opt"},
-      {"tarball": "config-agent/config-agent-*.tar.bz2", "sysroot": "/opt/smartdc"}
+      {"tarball": "config-agent/config-agent-*.tar.gz", "sysroot": "/opt/smartdc"}
     ]
   },
 
@@ -992,9 +992,9 @@ cat <<EOF
       "config-agent"
     ],
     "tarballs": [
-      {"tarball": "registrar/registrar-pkg-*.tar.bz2", "sysroot": "/"},
+      {"tarball": "registrar/registrar-pkg-*.tar.gz", "sysroot": "/"},
       {"tarball": "amon/amon-agent-*.tgz", "sysroot": "/opt"},
-      {"tarball": "config-agent/config-agent-*.tar.bz2", "sysroot": "/opt/smartdc"}
+      {"tarball": "config-agent/config-agent-*.tar.gz", "sysroot": "/opt/smartdc"}
     ]
   },
 
@@ -1019,9 +1019,9 @@ cat <<EOF
       "config-agent"
     ],
     "tarballs": [
-      {"tarball": "registrar/registrar-pkg-*.tar.bz2", "sysroot": "/"},
+      {"tarball": "registrar/registrar-pkg-*.tar.gz", "sysroot": "/"},
       {"tarball": "amon/amon-agent-*.tgz", "sysroot": "/opt"},
-      {"tarball": "config-agent/config-agent-*.tar.bz2", "sysroot": "/opt/smartdc"}
+      {"tarball": "config-agent/config-agent-*.tar.gz", "sysroot": "/opt/smartdc"}
     ]
   },
 
@@ -1047,9 +1047,9 @@ cat <<EOF
       "config-agent"
     ],
     "tarballs": [
-      {"tarball": "registrar/registrar-pkg-*.tar.bz2", "sysroot": "/"},
+      {"tarball": "registrar/registrar-pkg-*.tar.gz", "sysroot": "/"},
       {"tarball": "amon/amon-agent-*.tgz", "sysroot": "/opt"},
-      {"tarball": "config-agent/config-agent-*.tar.bz2", "sysroot": "/opt/smartdc"}
+      {"tarball": "config-agent/config-agent-*.tar.gz", "sysroot": "/opt/smartdc"}
     ]
   },
 
@@ -1074,9 +1074,9 @@ cat <<EOF
       "config-agent"
     ],
     "tarballs": [
-      {"tarball": "registrar/registrar-pkg-*.tar.bz2", "sysroot": "/"},
+      {"tarball": "registrar/registrar-pkg-*.tar.gz", "sysroot": "/"},
       {"tarball": "amon/amon-agent-*.tgz", "sysroot": "/opt"},
-      {"tarball": "config-agent/config-agent-*.tar.bz2", "sysroot": "/opt/smartdc"}
+      {"tarball": "config-agent/config-agent-*.tar.gz", "sysroot": "/opt/smartdc"}
     ]
   },
 
@@ -1103,9 +1103,9 @@ cat <<EOF
       "config-agent"
     ],
     "tarballs": [
-      {"tarball": "registrar/registrar-pkg-*.tar.bz2", "sysroot": "/"},
+      {"tarball": "registrar/registrar-pkg-*.tar.gz", "sysroot": "/"},
       {"tarball": "amon/amon-agent-*.tgz", "sysroot": "/opt"},
-      {"tarball": "config-agent/config-agent-*.tar.bz2", "sysroot": "/opt/smartdc"}
+      {"tarball": "config-agent/config-agent-*.tar.gz", "sysroot": "/opt/smartdc"}
     ]
   },
 
@@ -1130,9 +1130,9 @@ cat <<EOF
       "config-agent"
     ],
     "tarballs": [
-      {"tarball": "registrar/registrar-pkg-*.tar.bz2", "sysroot": "/"},
+      {"tarball": "registrar/registrar-pkg-*.tar.gz", "sysroot": "/"},
       {"tarball": "amon/amon-agent-*.tgz", "sysroot": "/opt"},
-      {"tarball": "config-agent/config-agent-*.tar.bz2", "sysroot": "/opt/smartdc"}
+      {"tarball": "config-agent/config-agent-*.tar.gz", "sysroot": "/opt/smartdc"}
     ]
   },
 
@@ -1158,10 +1158,10 @@ cat <<EOF
       "config-agent"
     ],
     "tarballs": [
-      {"tarball": "registrar/registrar-pkg-*.tar.bz2", "sysroot": "/"},
-      {"tarball": "mackerel/mackerel-pkg-*.tar.bz2", "sysroot": "/"},
+      {"tarball": "registrar/registrar-pkg-*.tar.gz", "sysroot": "/"},
+      {"tarball": "mackerel/mackerel-pkg-*.tar.gz", "sysroot": "/"},
       {"tarball": "amon/amon-agent-*.tgz", "sysroot": "/opt"},
-      {"tarball": "config-agent/config-agent-*.tar.bz2", "sysroot": "/opt/smartdc"}
+      {"tarball": "config-agent/config-agent-*.tar.gz", "sysroot": "/opt/smartdc"}
     ]
   },
 
@@ -1187,9 +1187,9 @@ cat <<EOF
       "config-agent"
     ],
     "tarballs": [
-      {"tarball": "registrar/registrar-pkg-*.tar.bz2", "sysroot": "/"},
+      {"tarball": "registrar/registrar-pkg-*.tar.gz", "sysroot": "/"},
       {"tarball": "amon/amon-agent-*.tgz", "sysroot": "/opt"},
-      {"tarball": "config-agent/config-agent-*.tar.bz2", "sysroot": "/opt/smartdc"}
+      {"tarball": "config-agent/config-agent-*.tar.gz", "sysroot": "/opt/smartdc"}
     ]
   },
 
@@ -1212,9 +1212,9 @@ cat <<EOF
       "config-agent"
     ],
     "tarballs": [
-      {"tarball": "registrar/registrar-pkg-*.tar.bz2", "sysroot": "/"},
+      {"tarball": "registrar/registrar-pkg-*.tar.gz", "sysroot": "/"},
       {"tarball": "amon/amon-agent-*.tgz", "sysroot": "/opt"},
-      {"tarball": "config-agent/config-agent-*.tar.bz2", "sysroot": "/opt/smartdc"}
+      {"tarball": "config-agent/config-agent-*.tar.gz", "sysroot": "/opt/smartdc"}
     ]
   },
 
@@ -1242,10 +1242,10 @@ cat <<EOF
       "config-agent"
     ],
     "tarballs": [
-      {"tarball": "minnow/minnow-pkg-*.tar.bz2", "sysroot": "/"},
-      {"tarball": "registrar/registrar-pkg-*.tar.bz2", "sysroot": "/"},
+      {"tarball": "minnow/minnow-pkg-*.tar.gz", "sysroot": "/"},
+      {"tarball": "registrar/registrar-pkg-*.tar.gz", "sysroot": "/"},
       {"tarball": "amon/amon-agent-*.tgz", "sysroot": "/opt"},
-      {"tarball": "config-agent/config-agent-*.tar.bz2", "sysroot": "/opt/smartdc"}
+      {"tarball": "config-agent/config-agent-*.tar.gz", "sysroot": "/opt/smartdc"}
     ]
   },
 
@@ -1269,9 +1269,9 @@ cat <<EOF
       "config-agent"
     ],
     "tarballs": [
-      {"tarball": "registrar/registrar-pkg-*.tar.bz2", "sysroot": "/"},
+      {"tarball": "registrar/registrar-pkg-*.tar.gz", "sysroot": "/"},
       {"tarball": "amon/amon-agent-*.tgz", "sysroot": "/opt"},
-      {"tarball": "config-agent/config-agent-*.tar.bz2", "sysroot": "/opt/smartdc"}
+      {"tarball": "config-agent/config-agent-*.tar.gz", "sysroot": "/opt/smartdc"}
     ]
   },
 
@@ -1295,9 +1295,9 @@ cat <<EOF
       "config-agent"
     ],
     "tarballs": [
-      {"tarball": "registrar/registrar-pkg-*.tar.bz2", "sysroot": "/"},
+      {"tarball": "registrar/registrar-pkg-*.tar.gz", "sysroot": "/"},
       {"tarball": "amon/amon-agent-*.tgz", "sysroot": "/opt"},
-      {"tarball": "config-agent/config-agent-*.tar.bz2", "sysroot": "/opt/smartdc"}
+      {"tarball": "config-agent/config-agent-*.tar.gz", "sysroot": "/opt/smartdc"}
     ]
   },
 
@@ -1323,9 +1323,9 @@ cat <<EOF
       "config-agent"
     ],
     "tarballs": [
-      {"tarball": "registrar/registrar-pkg-*.tar.bz2", "sysroot": "/"},
+      {"tarball": "registrar/registrar-pkg-*.tar.gz", "sysroot": "/"},
       {"tarball": "amon/amon-agent-*.tgz", "sysroot": "/opt"},
-      {"tarball": "config-agent/config-agent-*.tar.bz2", "sysroot": "/opt/smartdc"}
+      {"tarball": "config-agent/config-agent-*.tar.gz", "sysroot": "/opt/smartdc"}
     ]
   },
 
@@ -1360,9 +1360,9 @@ cat <<EOF
       "https://download.joyent.com/pub/build/sdcnode"
     ],
     "tarballs": [
-      {"tarball": "registrar/registrar-pkg-*.tar.bz2", "sysroot": "/"},
+      {"tarball": "registrar/registrar-pkg-*.tar.gz", "sysroot": "/"},
       {"tarball": "amon/amon-agent-*.tgz", "sysroot": "/opt"},
-      {"tarball": "config-agent/config-agent-*.tar.bz2", "sysroot": "/opt/smartdc"}
+      {"tarball": "config-agent/config-agent-*.tar.gz", "sysroot": "/opt/smartdc"}
     ]
   },
 
@@ -1418,9 +1418,9 @@ cat <<EOF
       "registrar"
     ],
     "tarballs": [
-      {"tarball": "registrar/registrar-pkg-*.tar.bz2", "sysroot": "/"},
+      {"tarball": "registrar/registrar-pkg-*.tar.gz", "sysroot": "/"},
       {"tarball": "amon/amon-agent-*.tgz", "sysroot": "/opt"},
-      {"tarball": "config-agent/config-agent-*.tar.bz2", "sysroot": "/opt/smartdc"}
+      {"tarball": "config-agent/config-agent-*.tar.gz", "sysroot": "/opt/smartdc"}
     ]
   },
 
@@ -1446,8 +1446,8 @@ cat <<EOF
     ],
     "tarballs": [
       {"tarball": "amon/amon-agent-*.tgz", "sysroot": "/opt"},
-      {"tarball": "config-agent/config-agent-*.tar.bz2", "sysroot": "/opt/smartdc"},
-      {"tarball": "registrar/registrar-pkg-*.tar.bz2", "sysroot": "/"}
+      {"tarball": "config-agent/config-agent-*.tar.gz", "sysroot": "/opt/smartdc"},
+      {"tarball": "registrar/registrar-pkg-*.tar.gz", "sysroot": "/"}
     ]
   },
 
@@ -1468,9 +1468,9 @@ cat <<EOF
         "registrar"
     ],
     "tarballs": [
-      {"tarball": "registrar/registrar-pkg-*.tar.bz2", "sysroot": "/"},
+      {"tarball": "registrar/registrar-pkg-*.tar.gz", "sysroot": "/"},
       {"tarball": "amon/amon-agent-*.tgz", "sysroot": "/opt"},
-      {"tarball": "config-agent/config-agent-*.tar.bz2", "sysroot": "/opt/smartdc"}
+      {"tarball": "config-agent/config-agent-*.tar.gz", "sysroot": "/opt/smartdc"}
     ]
   },
 
@@ -1496,7 +1496,7 @@ cat <<EOF
     ],
     "tarballs": [
       {"tarball": "amon/amon-agent-*.tgz", "sysroot": "/opt"},
-      {"tarball": "config-agent/config-agent-*.tar.bz2", "sysroot": "/opt/smartdc"}
+      {"tarball": "config-agent/config-agent-*.tar.gz", "sysroot": "/opt/smartdc"}
     ]
   },
 
